docs/develop/dapps/ton-connect/message-builders.mdx

```diff
@@ -598,7 +598,7 @@ The `body` for Jetton Burn([TEP-74](https://github.com/ton-blockchain/TEPs/blob/
     const body = beginCell()
         .storeUint(0x595f07bc, 32)                // jetton burn op code
         .storeUint(0, 64)                         // query_id:uint64
-        .storeCoins(1000000)                      // amount:(VarUInteger 16) -  Jetton amount in decimal
+        .storeCoins(toNano("0.001"))                      // amount:(VarUInteger 16) -  Jetton amount in decimal
```

Comment:
Let's add clarification, that decimal size individually specified for every jetton by its master contract